### PR TITLE
Update 1.48.10. JB#56372

### PR DESCRIPTION
**Pull request by pvuorela**
@mlehtima @abranson @Tomin1

### CODE REVIEW

File: rpm/pango.spec [hunk 1]
```diff
@@ -1,11 +1,10 @@
 Name:       pango
 Summary:    System for layout and rendering of internationalized text
-Version:    1.42.3
+Version:    1.48.10
 Release:    1
-Group:      System/GUI/GNOME
 License:    LGPLv2+
 URL:        http://www.pango.org
-Source0:    http://download.gnome.org/sources/pango/1.42/pango-%{version}.tar.xz
+Source0:    http://download.gnome.org/sources/pango/1.48/pango-%{version}.tar.xz
```
Comment (mlehtima) [inline rpm/pango.spec]: This probably could be just %{name}-%{version}.tar.xz which is the way we often have the source in spec files.

Comment (pvuorela) [reply rpm/pango.spec]: Adjusted though not sure if it matters much. Pango was even one character shorter :)

Comment (Tomin1) [reply rpm/pango.spec]: I think what @mlehtima meant was that you don't need to have that url there. Just `Source0: %{name}-%{version}.tar.xz` would be fine.

Comment (mlehtima) [reply rpm/pango.spec]: Yes, I meant that it should have Source0: %{name}-%{version}.tar.xz

Comment (pvuorela) [reply rpm/pango.spec]: Got it finally :) Changed.

**Review by mlehtima — APPROVED** [review]

**Pull request merged**